### ISSUE TITLE
Cover counter not updating [Version 0.6]

### ISSUE THREAD

**Issue by Nokius** [SfietKonstantin/harbour-twablet#59]
Cover counter not updating when refreshing via Cover Icon

**Comment by SfietKonstantin** [comment]
Intersting ...

**Comment by Nokius** [comment]
indeed

**Comment by Nokius** [comment]
some for the bar if refresh is done Search; Home and Mention shows not the new tweets :(

**Comment by SfietKonstantin** [comment]
Yep, got this issue. One of the first to be fixed in 0.7 I guess

**Comment by Nokius** [comment]
Great! Thanks

**Comment by Nokius** [comment]
fixed in 0.6.4 but now I have again this issue with missing streams [no issue ticket for that] :(

**Comment by SfietKonstantin** [comment]
Missing streams ?

**Comment by Nokius** [comment]
stream == line of content [tweets]

**Comment by SfietKonstantin** [comment]
There are a lot of regressions in 0.7dev (ie 0.6.X, X > 1) please fill all bugs you find so that I can track them

**Comment by Nokius** [comment]
ok, but this effect is not new we had this before (when the counting was working) but it did happen when the counting was broken
